app/models/user.rb

```diff
@@ -2,4 +2,8 @@ class User < ApplicationRecord
   has_many :posts
```

Comment:
- [ ] Since the foreign key has a different name (**author**) with the **user** table, it would be nice to specify it in the `app/models/user.rb` model.
Example: `has_many :posts, foreign_key: 'author_id'`